ci/build39.sh

```diff
@@ -6,14 +6,7 @@ pip install --no-deps -U pip wheel setuptools
 pip install python-dateutil pytz pytest pytest-xdist hypothesis
 pip install cython --pre # https://github.com/cython/cython/issues/3395
 
-# Alternative for getting numpy:
-# pip install --pre -f https://7933911d6844c6c53a7d-47bd50c35cd79bd838daf386af554a83.ssl.cf2.rackcdn.com/ numpy
-git clone https://github.com/numpy/numpy
-cd numpy
-python setup.py build_ext --inplace
-python setup.py install
-cd ..
-rm -rf numpy
+pip install --pre -f https://7933911d6844c6c53a7d-47bd50c35cd79bd838daf386af554a83.ssl.cf2.rackcdn.com numpy
```

Comment:
this might not be the right address iirc this changed
maybe have to look on the numpy issue list

Reply:
Its the same address we use in the numpy dev build? - https://github.com/pandas-dev/pandas/blob/master/ci/deps/azure-37-numpydev.yaml#L19

Reply:
Its the same address we use in the numpy dev build? - https://github.com/pandas-dev/pandas/blob/master/ci/deps/azure-37-numpydev.yaml#L19

Reply:
these might be older: 


https://anaconda.org/scipy-wheels-nightly/numpy/files
i think is the new location (includes scipy as well)

let’s update

Reply:
Ahh nice - thanks wasn't aware! Will update the numpy dev build too in a separate PR.

thanks!